util/strings.go

```diff
@@ -59,3 +59,25 @@ func EscapeJQLReservedChars(in string) string {
 	in = strings.Replace(in, `:`, `\\:`, -1)
 	return in
 }
+
+// GetSetDifference returns the elements in A that are not in B
+func GetSetDifference(a, b []string) (difference []string) {
+	setB := make(map[string]bool)
+	setDifference := make(map[string]bool)
```

Comment:
I tend to use map[string]struct{} for sets and then use the second "ok" value to check membership rather than bools

Reply:
Sure, let's save that space.